posts/azure-seen-by-developers.md

```diff
@@ -0,0 +1,190 @@
+
+![Figure 18 : Access keys for the connection to the storage](https://github.com/aversaveau/blog-usa/blob/azureSeenByDevs/images/2021/03/azure_seen_by_devs_18.png)
+
+This view provides the two key elements for the connection of a web app to an Azure storage: the Account name and the Account key. These two pieces of information will be entered in the application's environment variables and used by the Azure library when connecting to the storage of the environment concerned.
+
+![Figure 19 : the Shared access signature to obtain rights on the storage](https://github.com/aversaveau/blog-usa/blob/azureSeenByDevs/images/2021/03/azure_seen_by_devs_19.png)
+
+This view allows to generate access tokens (which are in fact URIs) providing restricted access rights to the storage and limited in time in order to secure the data stored there. Without a signature, a web app will not be able to access the resources of its storage. As far as signature generation is concerned, it is possible to generate a URI with read and write rights on all the storage resources for an unlimited period of time, but be careful about security. It is also possible to provide a read-only access token for one hour. Everything depends on the need.
+
+### Containers
+
+![Figure 20 : the storage containers](https://github.com/aversaveau/blog-usa/blob/azureSeenByDevs/images/2021/03/azure_seen_by_devs_20.png)
+
+This view lists all the containers created in the storage. It allows to create new ones, to modify them after creation and to delete them. Let's simulate the creation of a new container.
+
+![Figure 21 : creation of a new Blob container](https://github.com/aversaveau/blog-usa/blob/azureSeenByDevs/images/2021/03/azure_seen_by_devs_21.png)
+
+This view is the interface for creating a new container. It is very simple, requiring only the name of the new container and the access level. When you click on the drop-down list, the three options shown in Figure 22 are displayed.
+
+![Figure 22 : editing the access level of a Blob container](https://github.com/aversaveau/blog-usa/blob/azureSeenByDevs/images/2021/03/azure_seen_by_devs_22.png)
+
+The three levels of access to a container are _Private_, _Blob_ and _Container_. Configured on Private, a token is required to access the resources of the container. Configured on Blob, it is possible to access the resources hosted in the storage without an access token. Finally, configured on Container, it is possible to access the contents of the containers without access token.
+
+![Figure 23 : editing the access level of a Container](https://github.com/aversaveau/blog-usa/blob/azureSeenByDevs/images/2021/03/azure_seen_by_devs_23.png)
+
+It is possible to change the accessibility level of a container even after it has been created by right-clicking on it, which can be very useful.
```

Comment:
It is possible to change the accessibility level of a container, even after it has been created.  Simply right-click on it.